packages/reporting/src/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/app';
+import reportWebVitals from './reportWebVitals';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root'),
+);
+
+// NOTE: There is no point to have the app work offline as all the data must come from rmf/romi and
+// cannot be cached.
```

Comment:
I guess this doesn't hold true anymore, even though the data comes from rmf, the assets can still be cached. We could experiment with offline first in the future but for now just remove this note.

Reply:
Got it! Thanks